landoapi/transplants.py

```diff
@@ -211,9 +211,9 @@ def warning_previously_landed(*, revision, diff, **kwargs):
     diff_id = PhabricatorClient.expect(diff, "id")
 
     landed_transplant = (
-        Transplant.revisions_query([revision_id])
-        .filter_by(status=TransplantStatus.landed)
-        .order_by(Transplant.updated_at.desc())
+        LandingJob.revisions_query([revision_id])
+        .filter_by(status=LandingJobStatus.LANDED)
+        .order_by(LandingJob.updated_at.desc())
         .first()
     )
```

Comment:
Did we have entries for each landing in both tables (`LandingJob` and `Transplant`) before? I understand how this is a replacement from the logic side of things, but I don't get how this would work if we didn't have both tables up to date with the latest landings.

Reply:
> Did we have entries for each landing in both tables (`LandingJob` and `Transplant`) before?

Once a repo was switched over, we stopped creating `Transplant` entries. This backwards compatibility was really in place for a very small number of stacks that had parts of them landed with the old system, and other parts landed with the new system.

This particular line change actually fixes a bug, as this warning was not doing anything for non-legacy landings. I will add a note about that in the description.

Reply:
This is how the warning should look like, and how it will look like with this fix.

<img width="344" alt="image" src="https://user-images.githubusercontent.com/2043828/224825914-4a708348-063f-465b-9fa4-5ad7aa288e30.png">